readthedocs/oauth/services/bitbucket.py

```diff
@@ -214,5 +214,5 @@ def setup_webhook(self, project):
             log.error('Bitbucket webhook creation failed for project: %s',
                       project)
             log.debug('Bitbucket webhook creation failure response: %s',
-                      dict(resp))
+                      resp.content)
```

Comment:
`resp.json()` is more correct. Also, feel free to drop this, I have a fix in my branch.